src/test/java/arb/logic/commands/project/MarkProjectCommandTest.java

```diff
@@ -0,0 +1,71 @@
+package arb.logic.commands.project;
+
+import static arb.logic.commands.CommandTestUtil.assertCommandFailure;
+import static arb.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static arb.testutil.TypicalAddressBook.getTypicalAddressBook;
+import static arb.testutil.TypicalIndexes.INDEX_FIRST_CLIENT;
+import static arb.testutil.TypicalIndexes.INDEX_SECOND_CLIENT;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import arb.commons.core.Messages;
+import arb.commons.core.index.Index;
+import arb.model.ListType;
+import arb.model.Model;
+import arb.model.ModelManager;
+import arb.model.UserPrefs;
+import arb.model.project.Project;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code MarkProjectCommand}.
+ */
+class MarkProjectCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    void execute_validIndexUnfilteredList_success() {
+        Project projectToMark = model.getFilteredProjectList().get(INDEX_FIRST_CLIENT.getZeroBased());
+        MarkProjectCommand markProjectCommand = new MarkProjectCommand(INDEX_FIRST_CLIENT);
+        String expectedMessage = String.format(MarkProjectCommand.MESSAGE_MARK_PROJECT_SUCCESS, projectToMark);
+        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        projectToMark.markAsDone();
```

Comment:
Since you're utilising the same `projectToMark` object here, it will already have had its status set to `DONE` before calling `MarkProjectCommand`. I think you can make a copy of projectToMark `projectToMarkCopy` using `new ProjectBuilder(projectToMark).build();` and use that for `expectedModel` instead.

Reply:
okay!